crates/oxc_mangler/src/lib.rs

```diff
@@ -228,6 +244,29 @@ impl Mangler {
         frequencies.sort_unstable_by_key(|x| std::cmp::Reverse(x.frequency));
         frequencies
     }
+
+    fn collect_exported_symbols(program: &Program) -> (FxHashSet<CompactStr>, FxHashSet<SymbolId>) {
+        program
+            .body
+            .iter()
+            .filter_map(|statement| {
+                let Statement::ExportNamedDeclaration(v) = statement else { return None };
+                v.declaration.as_ref()
+            })
+            .flat_map(|decl| {
+                if let Declaration::VariableDeclaration(decl) = decl {
+                    itertools::Either::Left(
+                        decl.declarations
+                            .iter()
+                            .filter_map(|decl| decl.id.get_binding_identifier()),
+                    )
+                } else {
+                    itertools::Either::Right(decl.id().into_iter())
+                }
+            })
+            .map(|id| (id.name.to_compact_str(), id.symbol_id()))
+            .collect()
+    }
```

Comment:
The best approach is to have a symbol flag for the exports 🤔 but there aren't any...